certora/specs/Health.spec

```diff
@@ -75,6 +78,44 @@ filtered {
     assert !priceChanged => stillHealthy;
 }
 
+// The general case of liquidate for the stayHealthy rule
+//   - the user checked is not the borrower,
+//   - the market of the liquidation is the market of the user,
+//   - there is still some borrow on the market after liquidation,
+//   - there was no bad debt realization.
+rule stayHealthyLiquidate(env e, MorphoHarness.MarketParams marketParams, address borrower, uint256 seizedAssets, bytes data) {
+    MorphoHarness.Id id = libId(marketParams);
+    address user;
+
+    priceChanged = false;
```

Comment:
why do we need to do this + `require !priceChanged;`

Reply:
If the price changed then every account might become unhealthy as the value of the collateral can be 0. Note that here we are preventing the oracle to be updated in the middle of function calls (for example in the middle of liquidate)

Reply:
My question was: why do we set priceChanged to false **AND** require !priceChanged; Only the latter isn't enough?

Reply:
You are right, only the latter is enough. Removed in the latest version